force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -32,7 +32,17 @@ public with sharing class VariablesDataTypesOperators {
      * @return The sum of the two numbers, or null if either number is null.
      */
     public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
+       
+if(a == null || b == null){
```

Comment:
Be aware of the indenting here. Proper indenting makes it much easier to read the code later. There are commands and tools in vscode to help you with this.